### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by tsande16**
This PR will cleanup old documentation within the pass-support repository. It updates the top level README with a brief summary and adds URLS pointing to the new documentation site.

Related to https://github.com/eclipse-pass/main/pull/1081. When merged PR 1081 will close the https://github.com/eclipse-pass/main/issues/1035; however these PRs can be merged in any order.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
+# PASS Support
+
+PASS Support contains ancillary components that assist in the submission process and is composed of the Data Loaders, 
+Notification Services, and Deposit Services. The Data Loaders handle data ingestion from external systems like PubMed 
+Central, NIH, FIBI (JHU Grant Management System), transforming grant, journal, and manuscript submission data into a 
+standardized format within PASS. Deposit Services is responsible for the managing, packaging, and transfer of these 
+submissions to downstream repositories such as PubMed Central and institutional repositories. Notification Services 
+provide alerts and updates to relevant stakeholders based on submission workflows and events.
+
+You can find further documentation and details about the [Data Loaders](https://docs.eclipse-pass.org/developer-documentation/data-loaders),
```
[the remark this answers is not in the document]
Comment (tsande16) [reply README.md:10]: These are some good points. I'm getting some feedback on others PRs, so I'm going to compile these and we can talk about them at our PASS Status meeting.

[the remark this answers is not in the document]
Comment (tsande16) [reply README.md]: Done.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,15 @@
-# Introduction
-
-The pass-support repository contains components which sit outside pass-core.
-
+# PASS Support
```
[the remark this answers is not in the document]
Comment (tsande16) [reply README.md]: Done.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,22 @@
-# Introduction
+# PASS Support ![Pass Support](https://github.com/eclipse-pass/pass-support/actions/workflows/snapshot.yml/badge.svg)
 
-The pass-support repository contains components which sit outside pass-core.
+PASS Support contains ancillary components that assist in the submission process and is composed of the Data Loaders, 
+Notification Services, and Deposit Services. The Data Loaders handle data ingestion from external systems like PubMed 
+Central, NIH, FIBI (JHU Grant Management System), transforming grant, journal, and manuscript submission data into a
```
[the remark this answers is not in the document]
Comment (tsande16) [reply README.md]: Done.

**Review by rpoet-jh — APPROVED** [review]
Looks good.

**Review by markpatton — APPROVED** [review]
Looks good.

**Pull request merged**